queries/toml/highlights.scm

```diff
@@ -1,6 +1,6 @@
 ; Properties
 ;-----------
-(bare_key) @type
+(bare_key) @markup.heading
```

Comment:
Maybe could we change this to 
```
Maybe could we change this to
```suggestion
(table
  (bare_key)) @markup.heading
```
many color schemes have bolded headings, and it would be annoying for the bold attribute to carry over to regular properties. Shame the grammar defines them as the same node...

Reply:
Yeah, this is one of the reasons why I said this https://github.com/nvim-treesitter/nvim-treesitter/pull/7108#issuecomment-2323113344 , the refactor that I have in my mind solves this.

Reply:
I think it would be better to leave it as is in this PR and solve it in the refactor since there is more special cases to this problem (dotted keys and tables array headings).

Reply:
I had planned on tackling this, since I spent some time reading the TOML specs. As far as I can tell, headings should _not_ be marked as headings here, since they are in fact tables (i.e., common dotted keys). So my solution would be to remove this highlight entirely and mark _all_ keys -- whether in pairs or in tables -- the same.

Reply:
Maybe leave this change out of this PR for now and make it part of the TOML refactor (where we can discuss specs and intention in a holistic fashion)?

Reply:
I'm fine with that.